From 82f415baeda2a29edf6a981216744d2c6beb201c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?An=C3=ADbal=20Lim=C3=B3n?= <anibal.limon@linaro.org>
Date: Fri, 28 Dec 2018 17:47:13 -0600
Subject: [PATCH] CMakeLists.txt: Disable testing
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

test needs gtest package, so not build the test dir

Signed-off-by: Roy Li <rongqing.li@windriver.com>
Signed-off-by: Aníbal Limón <anibal.limon@linaro.org>
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2bc1137bd..226724ba9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,9 +7,6 @@ cmake_minimum_required(VERSION 3.4.0)
 # Generic header locations
 include_directories(${PROJECT_BINARY_DIR}/include)
 
-
-enable_testing()
-
 option(WITH_DOC "Build documentation." OFF)
 option(USE_NLS "Localisation support." ON)
 
@@ -215,7 +212,6 @@ add_subdirectory(doc)
 add_subdirectory(dselect)
 add_subdirectory(ftparchive)
 add_subdirectory(methods)
-add_subdirectory(test)
 
 if (USE_NLS)
 add_subdirectory(po)
-- 
2.19.2

